### PR TITLE
release 2.0.1

### PR DESCRIPTION
**Pull request by kandu**
a minor release for clarifying some doc comments.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+2.0.1 (2019-06-04)
+------------------
+
+* Zed\_char: add options of combining marks for transform functions (#18)
```
Comment (pmetzger) [inline CHANGES.md]: Grammar here is a bit stilted. I'd recommend new phrasing but I don't understand what the sentence means so I can't.

Comment (kandu) [reply CHANGES.md]: Hmm, it seems my baby-level English really causes some trouble. I think I'll spend some time to improve it to, at least, child-level.

**Comment by kandu** [comment]
Thank you, pmetzger. I've rectified some typo/grammatical mistakes, but I'm not sure if it's now satisfied. I'll be very happy if you or other guys would rectify, polish or rewrite my commits if it's not grammatical.

**Comment by kandu** [comment]
Hi, @diml , is there anything else needs to be improved before the new minor release?
I think your approval is a necessary trigger to start a release process.

**Comment by pmetzger** [comment]
Looks good to me. Merge it!

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+2.0.1 (2019-06-04)
+------------------
+
+* Zed\_char: add an `indv_combining` option to the transforming functions(`of_uChars, zChars_of_uChars, of_utf8`) to determine whether extract individual combining marks from the parameter (#18)
```
Comment (pmetzger) [inline CHANGES.md]: Oops! Just realized there's a small problem here. This needs to be no more than 80 columns wide! Just reformat it.

**Comment by pmetzger** [comment]
If you reformat the line that's more than 80 columns wide, I think you are clear to merge this.

File: CHANGES.md (hunk 2 above)
Comment (kandu) [reply CHANGES.md]: Thank you, pmetzger. This sentence is CPL 80 formatted now.

**Comment by pmetzger** [comment]
Merged! :)

**Comment by diml** [comment]
> Hi, @diml , is there anything else needs to be improved before the new minor release?
I think your approval is a necessary trigger to start a release process.

I have only been following from far, so I don't want to hold off the release process. If there is a specific point on which you'd like my input, I'm happy to have a look. Otherwise, I'm happy to leave these decisions to people who are more actively involved with the day to day life of the project.

**Pull request merged**